### PR TITLE
pkg/archive: truncate timestamps when writing headers to avoid rounding, except when Format==PAX

### PR DESCRIPTION
**Pull request by nalind**
Deal with time rounding (per #238) by forcing truncation of timestamps.  Add Go 1.11 and Xenial to test matrices, and drop 1.8.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,48 +11,44 @@ services:
...
+    - GO_VERSION="stable"
+      DISTRO="ubuntu"
+
+    - GO_VERSION="1.11"
+      DISTRO="ubuntu"
+
+    - GO_VERSION="1.10"
+      DISTRO="ubuntu"
+
+    - GO_VERSION="1.9"
```
[the remark this answers is not in the document]
Comment (nalind) [reply .travis.yml]: It would make sense to do that, but I didn't want to replace what we were testing against all at once.

**Comment by nalind** [comment]
We're not testing on Xenial -- I didn't work out the right declarations to use to add it on top of the existing Trusty-based test runs.  I'd like to be testing the other drivers, too, but I guess that doesn't have to be done in this PR.

**Comment by rhatdan** [comment]
@giuseppe @TomSweeneyRedHat @vrothberg PTAL

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by giuseppe** [comment]
LGTM

**Pull request merged**